share/src/main/webapp/components/documentlibrary/documentlist.js

```diff
@@ -217,22 +203,24 @@
                // allowing drops to occur
                var fpanel = Dom.get("alf-filters"),
                offset = (fpanel ? parseInt(fpanel.style.width, 10) : 160);
-               
-               if (e.clientX > offset)
-               {
-                  // If the current x co-ordinate of the mouse pointer is greater than the width
-                  // of the tree element then we shouldn't move folder/documents.
-               }
-               else
-               {
-                  var payload =
-                  {
-                     elementId: id,
-                     callback: this.onDropTargetOwnerCallBack,
-                     scope: this
-                  }
-                  YAHOO.Bubbling.fire("dropTargetOwnerRequest", payload);           
-               }
+
+                // MNT-19306
+                // If the current x co-ordinate of the mouse pointer is NOT greater than the width
+                // of the tree element, or this is a breadcrumb, then we SHOULD move folder/documents.
+                if (e.clientX <= offset || Dom.hasClass(dropTarget, "crumb"))
+                {
+                   var payload =
+                       {
+                          elementId: id,
+                          callback: this.onDropTargetOwnerCallBack,
+                          scope: this
+                       }
+                   YAHOO.Bubbling.fire("dropTargetOwnerRequest", payload);
+                }
+                else {
```

Comment:
This may sound pedantic, but an empty else block does not make sense (I've seen it was in here since before the fix to this issue). Please remove it, and perhaps merge the comment from inside this else block with the comment before the if block.

Reply:
Thanks Alex, I've removed the empty else block shifted the comment.